src/main/kotlin/io/snyk/plugin/snykcode/core/PDU.kt

```diff
@@ -135,11 +136,15 @@ class PDU private constructor() : PlatformDependentUtilsBase() {
 
     override fun showLoginLink(project: Any?, message: String) {
 //        showError(message, project)
```

Comment:
Minor: This comment should go (I know, not part of this PR, but if you find time...)